demos/full/_env

```diff
@@ -8,5 +8,5 @@
 #HLX_FASTLY_AUTH=
 #HLX_FASTLY_NAMESPACE=
 
-# Github Bot
+# Github Bot & private repo support
```

Comment:
hmmm...I'm not sure if we can use the same token for both...we need to try it out...

Reply:
I agree. OTOH having to specifiy separate github tokens would be counter-intuitive and confusing.

Reply:
ok..let's see how this all works out